Einthusan/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
```

Comment:
https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

outdir has a / too much